pidemo/README.md

```diff
@@ -0,0 +1,81 @@
+# Pidemo
```

Comment:
Have a look at the other samples. We don't keep the generic readme for the samples but explain the purpose ...

Reply:
In the readme in the root directory we add a reference to every sample with a short explanation. Wondering in what category this falls. Feels a bit weird to me to have it in the root directory as it seems to lack context.